src/plugins/core/chart.ts

```diff
@@ -162,9 +160,13 @@ export class ChartPlugin extends CorePlugin<ChartState> implements ChartState {
         const sheetFiguresFrom = this.getters.getFigures(cmd.sheetId);
         for (const fig of sheetFiguresFrom) {
           if (fig.tag === "chart") {
-            const id = this.nextId.toString();
-            this.history.update("nextId", this.nextId + 1);
-            const chartDefinition = { ...deepCopy(this.chartFigures[fig.id]), id };
+            const figureIdBase = fig.id.split(FIGURE_ID_SPLITTER).pop();
+            const duplicateFigureId = [cmd.sheetIdTo, figureIdBase].join(FIGURE_ID_SPLITTER);
```

Comment:
What is more efficient ? Creating an array then join on this array, or 
```js
const duplicatedFigureId = `${cmd.sheetIdTo}${FIGURE_ID_SPLITTER}${figureIdBase}`
```
I do not have the answer, and I think we don't care, but if someone has the answer...

Reply:
the interpolation is somehow better I really don't think we will feel the difference though.

On 1000000 iteration:
join : 237 ms
interpolate: 178 ms

Reply:
Ok so we do not care :D